utils/build-script

```diff
@@ -26,30 +31,47 @@ from build_swift.build_swift import presets
 
 import six
 
-from swift_build_support.swift_build_support import (
-    diagnostics,
-    products,
-    shell,
-    targets,
-    workspace
-)
+from swift_build_support.swift_build_support import diagnostics
+from swift_build_support.swift_build_support import products
+from swift_build_support.swift_build_support import shell
+from swift_build_support.swift_build_support import targets
+from swift_build_support.swift_build_support import workspace
 from swift_build_support.swift_build_support.SwiftBuildSupport import (
     HOME,
     SWIFT_BUILD_ROOT,
     SWIFT_REPO_NAME,
-    SWIFT_SOURCE_ROOT,
-)
+    SWIFT_SOURCE_ROOT)
 from swift_build_support.swift_build_support.cmake import CMake
 from swift_build_support.swift_build_support.host_specific_configuration \
     import HostSpecificConfiguration
-from swift_build_support.swift_build_support.targets import \
-    StdlibDeploymentTarget
+from swift_build_support.swift_build_support.targets import (
+    StdlibDeploymentTarget)
 from swift_build_support.swift_build_support.toolchain import host_toolchain
 
-build_script_impl = os.path.join(
+
+# -----------------------------------------------------------------------------
+# Constants
+
+BUILD_SCRIPT_IMPL = os.path.join(
     SWIFT_SOURCE_ROOT, SWIFT_REPO_NAME, "utils", "build-script-impl")
 
 
+# -----------------------------------------------------------------------------
+# Helpers
+
+def clean_delay():
+    """Provide a short delay so accidentally invoked clean builds can be
+    canceled.
+    """
+
+    sys.stdout.write('Starting clean build in  ')
+    for i in range(3, 0, -1):
```

Comment:
Im not sure I find there to be value in the `range` over `(3, 2, 1)`

Reply:
I merely moved this chunk of code from the bottom of the file to the top section. I would rather not change its functionality in this PR.

Reply:
Sounds reasonable.